src/documenti/esterni/PdP/contenuti/consuntivo.tex

```diff
@@ -71,24 +78,24 @@ \subsubsection{Conclusioni}
 \subsection{Secondo periodo}
 
 \subsubsection{Consuntivo orario}
-\begin{table}[!ht]
+\begin{table}[H]
     \centering
     \begin{tabular}{|l|c|c|c|c|c|c|c|}
     \hline
     \textbf{Membro} & \multicolumn{1}{c|}{\textbf{RE}} & \multicolumn{1}{c|}{\textbf{AM}} & \multicolumn{1}{c|}{\textbf{AN}} & \multicolumn{1}{c|}{\textbf{PT}} & \multicolumn{1}{c|}{\textbf{PR}} & \multicolumn{1}{c|}{\textbf{VE}} & \multicolumn{1}{c|}{\textbf{Totale ore persona}} \\ \hline
-    \textit{Marco Mazzucato}  & -      & 2            & 2          & - & - & 2 (-1)        & 6 (-1)         \\ \hline
-    \textit{Marco Mamprin}    & -      & 2            & 1 (-0.5)   & - & - & 2 (-1)        & 5 (-1.5)       \\ \hline
-    \textit{Marko Vukovic}    & -      & 1.5          & 3 (-1)     & - & - & 3             & 7.5 (-1)       \\ \hline
-    \textit{Mattia Zanellato} & -      & -            & 3          & - & - & 3 (-0.5)      & 6 (-0.5)       \\ \hline
-    \textit{Emanuele Pase}    & -      & -            & 3          & - & - & 3 (-0.5)      & 6 (-0.5)       \\ \hline
-    \textit{Riccardo Contin}  & 4      & -            & 3 (-1)     & - & - & 1 (+0.5)      & 8 (-0.5)       \\ \hline
-    \textit{Lorenzo Onelia}   & -      & 2 (-2)       & 2 (-1)     & - & - & 3 (+0.5)      & 7 (-2.5)       \\ \hline
-    \textbf{Totale ore ruolo} & 4      & 7.5 (-2)     & 17 (-3.5)  & - & - & 17 (-2)       & 45.5 (-7.5)    \\ \hline
+    \textit{Marco Mazzucato}  & -      & 2            & 2          & - & - & 1 (-1)        & 5 (-1)         \\ \hline
+    \textit{Marco Mamprin}    & -      & 2            & 0.5 (-0.5)   & - & - & 1 (-1)        & 3.5 (-1.5)       \\ \hline
+    \textit{Marko Vukovic}    & -      & 1.5          & 2 (-1)     & - & - & 3             & 6.5 (-1)       \\ \hline
+    \textit{Mattia Zanellato} & -      & -            & 3          & - & - & 2.5 (-0.5)      & 5.5 (-0.5)       \\ \hline
+    \textit{Emanuele Pase}    & -      & -            & 3          & - & - & 2.5 (-0.5)      & 5.5 (-0.5)       \\ \hline
+    \textit{Riccardo Contin}  & 4      & -            & 2 (-1)     & - & - & 1.5 (+0.5)      & 7.5 (-0.5)       \\ \hline
+    \textit{Lorenzo Onelia}   & -      & - (-2)       & 1 (-1)     & - & - & 3.5 (+0.5)      & 4.5 (-2.5)       \\ \hline
```

Comment:
metterei "0 (-2)", in modo da evidenziare la differenza tra ruoli presenti nel preventivo (che hanno un numero di ore fatte) e ruoli non presenti nel preventivo (che hanno un "-" ad indicare che non è stata preventivata alcuna ora)